consensus/obcpbft/viewchange.go

```diff
@@ -413,6 +413,8 @@ func (instance *pbftCore) processNewView2(nv *NewView) error {
 		instance.persistQSet()
 	}
 
+	instance.updateViewChangeSeqNo()
```

Comment:
The simplest solution seems to me to explicitly set `seqNo` to `max(xSet`).  The correctness is more clear to me this way, as `seqNo` is used only by the primary to assign new sequence numbers, and the new primary will obviously include its own view change in the new view, I think this solves it somewhat cleanly.